R/explanation.R

```diff
@@ -50,6 +50,13 @@
 #' \code{j} and \code{k} both increased the value of the prediction, but that the effect of the k-th
 #' feature was larger than the j-th feature.
 #'
+#' The value in \code{dt[i, 1]} is the prediction value not assigned to any of the features
+#' (\ifelse{html}{\eqn{\phi}\out{<sub>0</sub>}}{\eqn{\phi_0}})
+#' It's equal for all observations and set by the user through the argument \code{prediction_zero}.
+#' In theory this value should be the expected prediction without conditioning on any features.
+#' Typically we set this value to the mean of the response in our training data, but other choices
```

Comment:
What is the difference between `mean of the response in out traning data` and `such as the mean of the predictions in the training data`?

Reply:
mean of reponse = mean(y)
mean of predictions  = mean(pred(x))

might change to `such as the mean of the predictions of the training data`

Reply:
Ah, of course.